v2/comparecomplyv1/compare_comply_v1.go

```diff
@@ -2181,34 +2206,34 @@ func (options *ConvertToHTMLOptions) SetHeaders(param map[string]string) *Conver
 // CreateBatchOptions : The CreateBatch options.
 type CreateBatchOptions struct {
 	// The Compare and Comply method to run across the submitted input documents.
-	Function *string `json:"function" validate:"required"`
+	Function *string `validate:"required"`
 
 	// A JSON file containing the input Cloud Object Storage credentials. At a minimum, the credentials must enable `READ`
 	// permissions on the bucket defined by the `input_bucket_name` parameter.
-	InputCredentialsFile io.ReadCloser `json:"input_credentials_file" validate:"required"`
+	InputCredentialsFile io.ReadCloser `validate:"required"`
```

Comment:
just want to confirm that we dont need param name here anymore

Reply:
@mediumTaj no - there is only one case where we need to keep the JSON tags and that is for the recognition options model in STT due to Websockets.